srcs/request_handling/FixedResponses.cpp

```
@@ -94,3 +94,21 @@ std::unique_ptr<ResponsePacket> created(const std::string &location)
 }
+
+std::unique_ptr<ResponsePacket> not_implemented()
+{
+	auto response_packet = std::make_unique<ResponsePacket>();
+	response_packet->set_status_code(501);
+	response_packet->set_status_message("Not Implemented");
+	response_packet->set_final_response();
+	return response_packet;
+}
+
+std::unique_ptr<ResponsePacket> method_not_allowed()
```

cppcheck

[style] The function 'method_not_allowed' is never used.

(unusedFunction)